manuscript/preprint.tex

```diff
@@ -20,41 +20,56 @@
 % Define style for urls
 \urlstyle{same}
 
-% Configure column separation and page geometry
-%\usepackage[hmargin=2cm, vmargin=3cm]{geometry}
-%\setlength{\columnsep}{2\columnsep}
-
 % Import variables
 \input{variables.tex}
 
+% Configure column separation and page geometry
+\usepackage[hmargin=2cm,vmargin=2.9cm]{geometry}
+
 
 % Define title, authors, affiliations and DOI
 % ===========================================
 \newcommand{\Author}{%
-    \SolerShort{} and \UiedaShort{}
+  \SolerShort{} \& \UiedaShort
+}
+\newcommand{\AuthorLong}{%
+  \Soler{} and \Uieda
 }
 \newcommand{\AuthorAffil}{%
     {\large
-        \Soler$^{1,2}$\orcidlink{\SolerORCID} and
-        \Uieda$^{3}$\orcidlink{\UiedaORCID}
+      \Soler$^{1,2}$ \orcidlink{\SolerORCID} and
+        \Uieda$^{3}$ \orcidlink{\UiedaORCID}
     }
     \\[0.4cm]
     {\small $^{1}$\CONICET (\SolerMail)} \\
     {\small $^{2}$\IGSV} \\
     {\small $^{3}$\Liverpool} \\
 }
+\newcommand{\PreprintDOI}{10.31223/X58G7C}
```

Comment:
I would move this to `variables.tex`.

Reply:
I've just move the DOI to `variables.tex` and also made some small refactors.

Reply:
👍🏾